config/base/manager/manager.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    control-plane: controller-manager
+    control-plane: operator-controller-controller-manager
```

Comment:
I think I would delete this label from the namespace. `catalogd` contributes this same namespace in it's manifest, which means we'll end up with conflicts or silent overwrites.

This is related to #967

Reply:
So, this is a minor change that can be done fairly quickly.

Reply:
[catalogd uses](https://github.com/operator-framework/catalogd/pull/292/files) `catalogd-controller-manager` and not `operator-controller-controller-manager`. So I am little confused here.

Reply:
@LalatenduMohanty I think what @joelanford is saying is that both catalogd and operator-controller would create this particular namespace based on recent manifest changes across both projects